charts/opentelemetry-operator/templates/admission-webhooks/operator-webhook.yaml

```diff
@@ -1,7 +1,27 @@
 {{- if and (.Values.admissionWebhooks.create) (not .Values.admissionWebhooks.certManager.enabled) }}
 {{- $altNames := list ( printf "%s-webhook.%s" (include "opentelemetry-operator.fullname" .) .Release.Namespace ) ( printf "%s-webhook.%s.svc" (include "opentelemetry-operator.fullname" .) .Release.Namespace ) -}}
-{{- $ca := genCA "opentelemetry-operator-operator-ca" 365 -}}
-{{- $cert := genSignedCert (include "opentelemetry-operator.fullname" .) nil $altNames 365 $ca -}}
+{{- $caCertEnc := "" }}
```

Comment:
Can we move this to a _helpers.tpl function to keep the manifest less busy?

Reply:
Agree, moved.